floodviz/hydrograph_utils.py

```diff
@@ -60,9 +67,21 @@ def req_hydrodata(sites, start_date, end_date, url_top):
         returns a dictonary with the requested data from the nwis service 
```

Comment:
Technically, yes. However since the string.join(siteIDs), the list will always contain just one dict with all series values. It just has to be wrapped in a list for NVD3. I suppose I could do this elsewhere.